### ISSUE TITLE
[Suggestion] Small Suggestions

### ISSUE THREAD

**Issue by inkdear** [An-Sar/PrimalCore#299]
Bunch of small suggestions, some very small, basically just everything I've thought of while planning a pack and messing around with the mod.

1. Primalcore machines/tools in more materials (mostly vanilla/common modded materials):
- Bronze, steel, bone workblades/other tools that are missing those materials. Perhaps other common modded metals, but bronze and steel seem like the most likely to be used
- Metal ladles would be handy and pretty realistic
- Brick, nether brick, and terracotta/hardened clay sariras/hibachis
- Ironwood (since it's fireproof and just providing a frame) and pure iron (barbecue!) hibachis
2. You can currently drink bottles of Urushi and Tannin with no ill effects, and drinking Ovis Atre milk doesn't seem to have an effect that I'm aware of. Perhaps hunger or poison for the first two, and a combination of a negative and positive effect for the latter? I could see the Atre milk causing you to feel very sick but also causing brief fire resistance or regeneration or something.
3. Perhaps optional config to apply a timer to the milking of the Ovis Atre so it can fit in better in packs with mods where livestock milking isn't infinite (animania, probably also fauna and ecology, hunger overhaul, etc.) Would also allow the Atre milk to be used more as a finite resource in tweakered recipes. (Personally I want to use it as a potion base.)
4. Option to directly right-click items into the cauldron and sariras.
5. Cauldron/ladle text indicating if the current contents will form a proper recipe.
6. It'd add a lot to the cauldron experience if the items in the cauldron could be physically seen in there. That'd be just. Mwah. Perfecto. Perfect.
7. In the same vein, perhaps the cauldron could subtly change in appearance depending on the heat level. Maybe it could bubble if it has liquid in it or get a slight hot metal tinge to the underside.

**Comment by darthvader45** [comment]
1. These are definitely good ideas, though you can use MagnumOpus for bone work blades until then.
2. Tannin is quite toxic, and has a negative effect on nutrition and digestion, so hunger would work for it, as for urushi it is less capable of harming someone through ingestion, and as such should remain as is.  Though, trying to get lacquer sticks out of a cooled cauldron full of urushi lacquer should cause a special potion effect that acts similarly to poison ivy, where the player itches, and can tend to cause damage to themselves over time by scratching at it (kinda like a lingering instant damage, but less damaging).  Would encourage players to use a ladle.  Ovis Atre could cause nausea and fire resistance.
3. Yeah, similar to how cows can only be milked every five minutes.  Maybe an added feature where if you tried to milk them during the cooldown, they get a bit hostile and just do a slight bit of damage?  Basically like saying, "Don't milk me yet, not ready!"
4.  This would be kind of redundant, as nobody really bothers to rebind the drop item key or use it for anything else.  Dropping in is fine as-is.
5.  Dirt stick could do this.
6.  Now that would be cool to see, perhaps as simple rendered textures, or a rendered inventory like most 3D furnace mods.
7.  I suppose that can be done, perhaps with similar code to TC's crucible.

**Comment by An-Sar** [comment]
3. Currently the Ovis are themselves a fluid tank, which holds 1 bucket or 4 bottles of milk, the milk gets replaced when they feed.

**Comment by inkdear** [comment]
3. Currently the Ovis are themselves a fluid tank, which holds 1 bucket or 4 bottles of milk, the milk gets replaced when they feed.
Ah, makes sense. I hadn't gotten up to the point of playing with them in survival and was just testing with bottles. Apologies.

4. This would be kind of redundant, as nobody really bothers to rebind the drop item key or use it for anything else. Dropping in is fine as-is.
I was thinking of it more as a QoL/experience-streamlining thing. When most of your other UI-less machines/grindstones/etc are click-based, stopping to throw things in can feel kind of jarring/fiddly (esp when you also need to count them.)

**Comment by darthvader45** [comment]
Yeah, (although I'm not a dev, that was just my personal opinion, my two cents)though it could be a bit more streamlined.  However, as long as you count up the amount needed prior to doing the recipe, you should be just fine.
